eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/responses/m2w/M2WebEwonEBDTag.java

```diff
@@ -32,7 +32,7 @@ public class M2WebEwonEBDTag {
    *
    * @since 1.0.0
    */
-  @CsvBindByName(column = "Description", required = true)
+  @CsvBindByName(column = "Description", required = false)
```

Comment:
here you are explicitly setting to false, other places you simply removed required parameter. Not sure if this matters.

Reply:
Will update -- the default is false, so there is no need to specify here.